packages/core/src/definitions/interfaces/prefix-command-global-options.ts

```diff
@@ -0,0 +1,29 @@
+export interface PrefixCommandGlobalOptions {
```

Comment:
I think this needs to be added to index file too [here](https://github.com/fjodor-rybakov/discord-nestjs/blob/master/packages/core/src/index.ts#L3)